src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form-big {
+  margin: 20px 0;
+  position: relative;
+}
+
+.form-small {
+  position: relative;
+}
+
+.form-big::before {
+  content: "";
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  align-items: center;
+  bottom: 26px;
+  width: 19px;
+  height: 19px;
+  background: url("images/Search.svg") no-repeat center / contain;
+}
+
+.form-small::before {
+  content: "";
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
+  width: 11px;
+  height: 11px;
+  background: url("images/Search.svg") no-repeat center / contain;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+}
+
+.input-big {
+  height: 70px;
+  padding: 0 0 0 62px;
+}
+
+.input-small {
+  height: 42px;
+  padding: 0 0 0 33px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
```

Comment:
Better to define in font family in body or html tag
And then use `inherit` value